builtin/game/chatcommands.lua

```diff
@@ -774,12 +774,28 @@ core.register_chatcommand("time", {
 })
 
 core.register_chatcommand("shutdown", {
+	params = "<delay_in_seconds(0..360)> | Leave blank for normal shutdown",
 	description = "shutdown server",
 	privs = {server=true},
 	func = function(name, param)
-		core.log("action", name .. " shuts down server")
-		core.request_shutdown()
-		core.chat_send_all("*** Server shutting down (operator request).")
+		if param ~= "" then
+			param = tonumber(param) or 0
+			if param > 0 or param < 360 then
```

Comment:
So... all real numbers? It's great that I can't shut down the server in 72+3i seconds, at least...

/shutdown -1

*crash*

Well, it does shut it down.